src/adapters/MeetingsAdapter.js

```diff
@@ -0,0 +1,90 @@
+import {throwError} from 'rxjs';
+
+import WebexAdapter from './WebexAdapter';
+
+/**
+ * This is a base class that defines the interface that maps meetings data.
+ * Developers that want to extend `MeetingsAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class MeetingsAdapter extends WebexAdapter {
+  /**
+   * A meeting object that allows users to have a WebRTC meeting.
+   *
+   * @typedef {Object} Meeting
+   * @property {string}  ID The meeting identifier.
+   * @property {string}  meetingTitle The title of the meeting.
+   * @property {Date}    meetingTimeStart The time and date of the start of the meeting.
+   * @property {Date}    meetingTimeEnd The time and date of the end of the meeting.
+   * @property {Boolean} localAudioMuted If the local user has muted their audio.
```

Comment:
I kinda don't like having `localAudioMuted`, `localVideoMuted` and `meetingJoined` mixed with the information of the meeting because they are states. Would it make sense to have `Meeting` object with the info and another `LocalMeetingState` with these states?
`LocalMeetingState` would be returned by `joinMeeting` method

Reply:
My thoughts were that the meeting state would be reported along with all of the meeting info in the meeting observer.